live/prod/data-stores/postgres/setup/membership.tf

```diff
@@ -0,0 +1,62 @@
+// Generate random username and password
+resource "random_string" "membership_db_user" {
+  length  = 16
+  special = false
+}
+
+resource "random_password" "membership_db_pass" {
+  length           = 20
+  special          = true
+  min_special      = 1
+  override_special = "~*^+"
+}
+
+// Store both in SSM
+resource "aws_ssm_parameter" "membership_db_user" {
+  name  = "/${local.environment}/services/membership/db_user"
+  type  = "String"
+  value = "fundraising_${random_string.membership_db_user.result}"
```

Comment:
`"fundraising_${random_string.membership_db_user.result}"` -> `"membership_${random_string.membership_db_user.result}"` no? Just making sure that this and the `fundraising_` at line 37 are intended

Reply:
yeah changing this requires us to recreate the user and I don't want to make database changes at this time. I'll change this and the database name which remains as "fundraising" in another PR.